unit_tests/test_reprepro_sign_helper.py

```diff
@@ -0,0 +1,40 @@
+from unittest import TestCase, mock
+
+from archive_auth_mirror.scripts.reprepro_sign_helper import (
+    gpg,
+    inline_sign,
+    detach_sign,
+)
+
+
+class GPGCallsTest(TestCase):
+
+    @mock.patch('subprocess.check_call')
+    def test_gpg(self, mock_check_call):
+        """The gpg function calls gpg."""
```

Comment:
I think we can do a better docstring than this.  How about "The gpg function invokes the system gpg binary, with appropriate arguments." ?